design/workitemtype.go

```diff
@@ -21,9 +21,15 @@ var fieldDefinition = a.Type("fieldDefinition", func() {
 	a.Description("A fieldDescription aggregates a fieldType and additional field metadata")
 	a.Attribute("required", d.Boolean)
 	a.Attribute("type", fieldType)
-
-	a.Required("required")
-	a.Required("type")
+	a.Attribute("label", d.String, "A label for the field that is shown in the UI", func() {
```

Comment:
@kwk Can we use a.Enum() here? I am asking this because `label` should not contain just any string. It is supposed to have limited number of values like -> "Title", "Description", "Created At", "Remote item", "Description", "Iteration", "Areas", "Assignees", "State"